docker/metrics/prod.yml

```
@@ -0,0 +1,7 @@
+version: "3.9"
+
+volumes:
+  grafana-storage:
+    external: true
+  influx-storage:
+    external: true
```

I have a new politic on this, we should add names to have project name in it (to be able to eventually deploy on same machine).

That means here:
```suggestion
volumes:
  grafana-storage:
    external: true
    name: ${COMPOSE_PROJECT_NAME}_grafana-storage
  influx-storage:
    external: true
    name: ${COMPOSE_PROJECT_NAME}_grafana-storage
```

and change in make_external_volumes target (see there).

---

Good idea, it avoids name conflicts